driver/test/java/io/crate/client/jdbc/integrationtests/CrateJDBCConnectionTest.java

```diff
@@ -129,8 +129,14 @@ private static void insertIntoTable() throws SQLException {
         preparedStatement.setObject(10, objectField);
         preparedStatement.setString(11, "1970-01-01");
         preparedStatement.setString(12, "127.0.0.1");
-        preparedStatement.setArray(13, connection.createArrayOf("varchar", new String[]{"a", "b", "c", "d"})); // TODO array support
-        preparedStatement.setArray(14, null);
+        preparedStatement.setArray(13, connection.createArrayOf("varchar", new String[]{"a", "b", "c", "d"}));
```

Comment:
1) Would it be best to just change the datatype strings in the info cache as a map? See here https://github.com/crate/pgjdbc/blob/crate/pgjdbc/src/main/java/org/postgresql/jdbc/TypeInfoCache.java#L70

2) Do you think this test should also be in `CrateJDBCConnectionTest` or in a separate testing class?